background/package.json

```diff
@@ -14,19 +14,21 @@
     "test": "mocha --exit **/*.test.js --timeout=50000"
   },
   "dependencies": {
-    "@google-cloud/firestore": "^3.0.0",
+    "@google-cloud/firestore": "^3.8.5",
     "@google-cloud/pubsub": "1.7.3",
     "@google-cloud/storage": "^4.0.0",
-    "body-parser": "^1.18.3"
+    "body-parser": "^1.18.3",
+    "firestore": "^1.1.6",
```

Comment:
That's interesting.  Is this different from `@google-cloud/firestore`

Reply:
Oops! That was a mistake. Thanks!